grazer/src/main/webapp/WEB-INF/views/person.jsp

```diff
@@ -29,7 +27,16 @@
 
 <h2>${concept.word}</h2>
 
-<p>${concept.description}</p>
+<p>
+${concept.description}
+</p>
+
+
+<c:forEach var="listVar" items="${concept.getEqualTo() }"> 
+	<c:if test="${listVar != '[]'}">
+    		<a href="${listVar}" target="_blank"> ${listVar}</a>
+    	</c:if>
+</c:forEach>
```

Comment:
Those should not show up just next to each other, each should be in one line. maybe add an icon in front of them (maybe font awesome external-link). and put some space between the links and the following list group.